src/rendering/vcTileRenderer.cpp

```diff
@@ -128,173 +149,157 @@ bool vcTileRenderer_ShouldLoadNode(vcQuadTreeNode *pNode)
   return pNode->renderInfo.tryLoad && pNode->touched && (pNode->renderInfo.loadStatus == vcNodeRenderInfo::vcTLS_InQueue);
 }
 
-void vcTileRenderer_LoadThread(void *pThreadData)
+void vcTileRenderer_PriorityInsert(udChunkedArray<vcTileLoadPacked> *pInput, vcTileRenderer *pRenderer, vcQuadTreeNode *pNode)
```

Comment:
This function does a lot of stuff that looks like it should be locked?

Reply:
This function is used in place of code that previously directly inserted, so in every instance where it's called it should already be locked

Reply:
It's usage is thread safe.

Reply:
This function doesn't appear to work correctly.